content/en/platform/corda/5.0-dev-preview-1/tutorials/building-cordapp/c5-basic-cordapp-flows.md

```diff
@@ -119,9 +121,9 @@ In this sample CorDapp, add these services:
 * `FlowIdentity`
```

Comment:
@peterli-r3 Could you please explain why do I want to add those services and add short description of what each service does?

Reply:
These service are the services that a Corda flow would need. In Corda 4, all of the services are automatically running behind the scene. However, in Corda 5, these services are invoked more agilely. You can only inject the services that the flow needs. This will make the flow more light weighted. 
---
short description: I cant. these should come from enginner.

Reply:
Added descs obtained from @conalsmith-r3